index.html

```diff
@@ -17,7 +17,7 @@
 				background: #242321;
 			}
 		</style>
-		<canvas id="canvas" width="1280px" height="720px" style="border: solid 1px black"></canvas>
+		<canvas id="canvas" width="1080px" height="720px" style="border: solid 1px black"></canvas>
```

Comment:
I think you wanted to change the height to 1080 and width to 1920

earlier it was 1280 now its 1080 making it "smaller"

Reply:
yeah i wanted to make it smaller, cuz ones the particles settle, there's too much empty space visible

Reply:
Now that I have added particle emitters so that will be a nonissue cause we have infinite particles

Reply:
yes that'll do it